libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/maneuver/ManeuverViewHelper.kt

```diff
@@ -0,0 +1,165 @@
+package com.mapbox.services.android.navigation.v5.internal.navigation.maneuver
```

Comment:
I am not sure if we want to add any `Kotlin` files in the legacy code. @Guardiola31337 what do you think?

Reply:
Disregard the above.